package.json

```diff
@@ -38,7 +38,7 @@
     "clean": "rimraf lib dist es coverage",
     "format": "prettier --write \"{src,test}/**/*.{js,ts}\" index.d.ts \"**/*.md\"",
     "format:check": "prettier --list-different \"{src,test}/**/*.{js,ts}\" index.d.ts \"**/*.md\"",
-    "lint": "eslint src test",
+    "lint": "eslint --ext js,ts src test",
```

Comment:
Needed to lint TypeScript files

Reply:
would you mind adding this to the examples:lint command also?

Reply:
Fixed. I didn't realize there was another ESLint command. Thanks for catching it.